libs/fixtures/src/index.ts

```diff
@@ -11,6 +11,10 @@ export {
   electionDefinition as electionGeneralDefinition,
   election as electionGeneral,
 } from './data/electionGeneral/election.json';
+export {
+  electionDefinition as electionMarkScanDiagnosticDefinition,
+  election as electionMarkScanDiagnostic,
+} from './data/electionGeneral/election.json';
```

Comment:
This is a clone of `electionGeneralDefinition` so that fixture can be changed without impact to the diagnostic